lib/modules/plugin_cmd/index.js

```diff
@@ -0,0 +1,64 @@
+let fs = require('./../../core/fs.js');
+let utils = require('./../../utils/utils.js');
+let async = require('async');
+class PluginCommand {
+  constructor(embark, config) {
+    this.embark = embark;
+    this.config = config;
+    this.embarkConfig = fs.readJSONSync(this.config.embarkConfigFile);
+    this.registerCommand();
+  }
+  registerCommand() {
+    const self = this;
+    self.embark.registerConsoleCommand((cmd, _options) => {
+      let cmdArray = cmd.split(' ');
+      cmdArray = cmdArray.filter(cmd => cmd.trim().length > 0);
+      let cmdName = cmdArray[0];
+      return {
+        match: () =>  cmdName === 'plugin',
+        process: (callback) => {
+          if(cmdArray.length < 3 || cmdArray[1] !== 'install' || typeof cmdArray[2] === 'undefined') {
+            return callback('invalid use of plugin command. Please use plugin install <package>');
+          }
+          let npmInstall = ['npm', 'install'];
+          if (cmdArray[2] === '--save') {
```

Comment:
I feel `--save` is useless. Just doing `plugin install nameOfPlugin` should be enough. Also, save can be at the end or before the package name, so we need to be careful.

Reply:
fixed in the latest commit